next/components/ui/PartnerCard/PartnerCard.tsx

```diff
@@ -11,10 +11,10 @@ export interface PartnerProps {
   featured?: boolean | null | undefined
 }
 
-export const Partner = ({ className, title, pageLink, logo, alt, featured }: PartnerProps) => {
+export const PartnerCard = ({ className, title, pageLink, logo, alt, featured }: PartnerProps) => {
```

Comment:
If you want to make it even cleaner, these are the suggestions:

- I thinks that PartnerCard can take the whole `partner` as prop. We don't use partners with meilisearch so it should be safe and much cleaner.
- I also find weird to pass "link label" by props. Pass just url, usually called `linkHref` as prop if not the whole  `partner`.
- In my opinion, we should't provide classNames override if not really needed so I would remove it to make it even cleaner.
- You may use Button with left icon instead of custom styled link.

All of this is our heritage from the old code we want to get rid off and clean it as much as possible.